tools/loader/Loader.h

```diff
@@ -45,6 +48,27 @@ bool profilingGraph();
 /// Parse/verify command line parameters.
 void parseCommandLine(int argc, char **argv);
 
+/// Loader extension interface from which to derive in order to extend the
+/// Loader driver object.
+class Loader;
+class ProtobufLoader;
+
+class LoaderExtension {
+public:
+  /// Called once after ONNX or Caffe2 model loading.
+  virtual void postModelLoad(Loader &, PlaceholderBindings &, ProtobufLoader &,
```

Comment:
Is this an optional call or not after ONNX or Caffe2 model loading?

Reply:
For all **registered** extensions (an extension is a class that derives from LoaderExtension), this method will be called. If for a particular extension, you don't plan any change after model load, then you just need to implement this method empty this extension. An extension may also decide to do something or not based on some command line options.